data/json/mutation_category.json

```diff
@@ -144,7 +144,7 @@
         "name" : "Medical","//" : "2-6 pain, same as Alpha--since specifically intended for medical applications.",
         "category" : "MEDICAL",
         "mutagen_message" : "You can feel the blood rushing through your veins and a strange, medicated feeling washes over your senses.",
-        "iv_message" : "You can feel the blood in your medication stream.  It's a strange feeling.",
+        "iv_message" : "You can feel the medication in your blood stream.  It's a strange feeling.",
```

Comment:
Ya but the way its written in game is not as good as that. "You can feel the blood in your medication stream.  It's a strange feeling." It looks more like a actual joke and sounds better without the "It's a strange feeling" part honestly.